xrootd/protocol/protocol.go

```diff
@@ -85,3 +87,63 @@ const (
 	// DataServer indicates whether this is a data server.
 	DataServer
 )
+
+// EntryStat holds the entry name and the entry stat information.
+type EntryStat struct {
```

Comment:
it would be great if we could have `EntryStat` implement `os.FileInfo`:
```
$> go doc os.FileInfo
type FileInfo interface {
	Name() string       // base name of the file
	Size() int64        // length in bytes for regular files; system-dependent for others
	Mode() FileMode     // file mode bits
	ModTime() time.Time // modification time
	IsDir() bool        // abbreviation for Mode().IsDir()
	Sys() interface{}   // underlying data source (can return nil)
}
    A FileInfo describes a file and is returned by Stat and Lstat.

```
or at least devise some way for it to interoperate with `os.FileInfo`.